client/extensions/wp-super-cache/state/cache/selectors.js

```diff
@@ -3,6 +3,10 @@
  */
 import { get } from 'lodash';
 
+function getCacheState( state ) {
+	return get( state, 'extensions.wpSuperCache.cache', {} );
```

Comment:
Bit of a nitpick, but I think you can drop the `get` here and access `state.extensions.wpSuperCache.cache` directly. Rationale -- If this selector is called, we can assume the wpSuperCache extension code and thus the `extensions.wpSuperCache` Redux substate to be present; and the `cache` reducer has an initial state which is set on startup.